docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -474,6 +474,16 @@ <h1>Samza Configuration Reference</h1>
                         These coordination utils are currently used for intermediate stream creation.
                     </td>
                 </tr>
+
+                <tr>
+                    <td class="property" id="job.store.path">job.store.path</td>
```

Comment:
Non-changelogged stores are stored in YARN container's workspace so that they can be clean up when the container is unused. Changelogged stores are kept separately and need something like state store GC to be cleaned up. Should this config be "job.logged.store.base.dir" or is the intention to move them all in the same place? If the latter, why?

Reply:
There's existing configuration `job.local.store.dir`, which is used by LocalStoreMonitor as a base directory for cleaning up state store.
I think we should unify those two configurations and not have duplicate configurations. Either delete the existing one or use that configuration instead. Update the relevant documentation .